scripts/post-install.sh

```diff
@@ -24,10 +24,8 @@ function install_chkconfig {
     chkconfig --add telegraf
 }
 
-id telegraf &>/dev/null
-if [[ $? -ne 0 ]]; then
-    grep "^telegraf:" /etc/group &>/dev/null
-    if [[ $? -ne 0 ]]; then
+if ! id telegraf &>/dev/null; then
+    if grep "^telegraf:" /etc/group &>/dev/null; then
```

Comment:
This reverses the logic from the previous code. Previous code executed evaluated as true if the match WASN'T found. This code evaluates as true if the match WAS found.